src/test/kotlin/org/jitsi/nlj/dtls/DtlsTest.kt

```diff
@@ -58,19 +59,23 @@ class DtlsTest : ShouldSpec() {
         val clientSender = ProtocolSender(dtlsClient)
         val clientReceiver = ProtocolReceiver(dtlsClient)
 
-        val pcapWriter = PcapWriter(logger, "/tmp/dtls-test.pcap")
+        val pcapWriter = PcapWriter(logger, if (pcapEnabled) "/tmp/dtls-test.pcap" else "/dev/null")
```

Comment:
Can we just make it nullable instead and do:
```
val pcapWriter = if (pcapEnabled) PcapWriter(logger, "/tmp/dtls-test.pcap") else null
```

And then below we can skip the if check and just do `pcapWriter?.processPacket(packetInfo)`